### PR TITLE
init passkey pallet

### LINKED ISSUE #2026: [Feature]: Passkey pallet scaffolding
### Feature Description
We would like to encapsulate all passkey related changes inside a pallet. 

# Acceptance Criteria
- Add a new pallet with bare minimum features
- Setup Test configuration
- Benchmark configuration
- Configure runtime and include this pallet with a feature flag
- Extend makefile to support building frequency with mentioned feature flag

### PR DESCRIPTION
**Pull request by aramikm**
# Goal
The goal of this PR is create an empty pallet for passkey

Closes #2026 

# Checklist
- [x] Chain spec updated
- [x] Tests added
- [x] Benchmarks added
- [x] Weights updated

### CODE REVIEW

File: runtime/frequency/src/lib.rs [hunk 1]
```diff
@@ -1176,6 +1183,9 @@ construct_runtime!(
 		Capacity: pallet_capacity::{Pallet, Call, Storage, Event<T>, FreezeReason} = 64,
 		FrequencyTxPayment: pallet_frequency_tx_payment::{Pallet, Call, Event<T>} = 65,
 		Handles: pallet_handles::{Pallet, Call, Storage, Event<T>} = 66,
+		// Currently enabled only under feature flag
+		#[cfg(any(not(feature = "frequency"), feature = "frequency-lint-check"))]
+		Passkey: pallet_passkey::{Pallet, Call, Storage, Event<T>} = 67,
```
Comment (wilwade) [inline runtime/frequency/src/lib.rs:1188]: ❤️

**Review by wilwade — APPROVED** [review]
Merge it

- ✅ Double checked for the feature flag for mainnet
- ✅ No impact on testnet
- ✅ Reviewed structure

**Review by saraswatpuneet — APPROVED** [review]
Lgtm

File: pallets/passkey/src/mock.rs [hunk 2]
```diff
@@ -0,0 +1,61 @@
+//! Mocks for the Time-release module.
```
Comment (mattheworris) [inline pallets/passkey/src/mock.rs:1]:
```suggestion
//! Mocks for the Passkey module.
```

**Review by mattheworris — APPROVED** [review]
🚀

**Review by JoeCap08055 — APPROVED** [review]

File: pallets/passkey/src/mock.rs (hunk 2 above)
Comment (aramikm) [reply pallets/passkey/src/mock.rs:1]: I'll address this in any future PR

**Pull request merged**